www/widgets/FontSizeWidget.js

```diff
@@ -0,0 +1,42 @@
+/* global GenWidget */
+
+class FontSizeWidget extends GenWidget {
```

Comment:
take a look at my chnages to `FontSizeWidget`. given the direction we've gone in w/GenWidget (having created that custom element) it's not longer necessary to setup that "data-update" event listener from the original design. So make sure to review what `FontSizeWidget` looks like now before u keep working on things as well (it's a bit simpler now).